src/main/java/org/redhat/appstudio/serviceprovider/service/storage/AccessToken.java

```diff
@@ -150,4 +151,26 @@ public String toString() {
         + '\''
         + '}';
   }
+
```

Comment:
Make sense to move (from model class) these adapters to something Vault specific? For example `VaultConverter`

Reply:
I don't like much the idea to writing classes which are completely rely on other class internals, and must know and follow it.
from the che-server practice, thats is bug-prone. let's think about better solution.

Reply:
these methods don't look good in POJO. what about https://stackoverflow.com/a/39544594/318536?